packages/jest-transform/src/__tests__/ScriptTransformer.test.ts

```diff
@@ -1109,11 +1109,11 @@ describe('ScriptTransformer', () => {
     /* eslint-disable sort-keys */
     const instrumentedCodeMap = {
       version: 3,
-      sources: ['banana.js'],
       names: ['content'],
-      mappings:
-        ';;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;AAeY;;;;;;;;;;AAfZA,OAAO',
+      sources: ['banana.js'],
       sourcesContent: ['content'],
+      mappings:
```

Comment:
order of keys has changed, but `AAeY` is also `IAeY`. just to verify @jridgewell, that's a correct change?

(same change in all the tests, fwiw)

Reply:
I don't fully understand source maps, but the reason is probably https://github.com/babel/babel/pull/14506

Reply:
That sounds likely 😀

Reply:
`A` (0) -> `I` (4) in the first VLQ field likely means we've skipped 4 space chars used for indentation.

Reply:
Cool, thanks!